sdk/communication/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.{cs,vb}]
+dotnet_diagnostic.CA1825.severity = none
```

Comment:
any reason we are suppressing this instead of fixing it?

Reply:
I have reverted this file.  It was for my local env only, this should not be a problem in the pipeline.